### ISSUE TITLE
slow maps on visualize PROD

### ISSUE THREAD

**Issue by Rdataflow** [visualize-admin/visualization-tool#1300]
**Describe the bug**
For some odd reason on visualize PROD the same GraphQL triggering the same SPARQL query has much longer response time.

**To Reproduce**

1. Go to https://visualize.admin.ch/de/v/lpFHNCtYBCaE?dataSource=Prod&flag__debug=true
2. Open debug panel, open GeoShapesByDimensionIri, see first SPARQL query
3. See it takes ~10s, while the same dataset viewed trough visualize TEST or INT triggering the same SPARQL query takes < 1s

**Expected behavior**
Equally fast response time as [TEST](https://test.visualize.admin.ch/v/pqOaldX6NutR?dataSource=Prod) or INT - or an explanation why timing would be biased on PROD...

**Screenshots or video**
![image](https://github.com/visualize-admin/visualization-tool/assets/26090254/0ca6b021-53df-4696-a881-7a7aff9783d4)

**Environment (please complete the following information):**
- Visualize environment and version: PROD v3.22.9 ([a09cdc3](https://github.com/visualize-admin/visualization-tool/commit/a09cdc30abf0744481670f6549f5766610073b1c))

**Additional context**
the very same SPARQL query is fired to the same geo endpoint. direct execution behaves fast. (<1s)
[SPARQL query on geo endpoint (identically fired from TEST, INT and PROD)](https://geo.ld.admin.ch/sparql/#query=%23pragma+describe.strategy+cbd%0A%23pragma+join.hash+off%0A%0A%0APREFIX+geo%3A+%3Chttp%3A%2F%2Fwww.opengis.net%2Font%2Fgeosparql%23%3E%0A%0ASELECT+%3Fgeometry+%3FwktString%0A%0AWHERE+%7B%0A%0A++++++++++++VALUES+%3Fgeometry+%7B%0A++++++++++++++%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1000%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F317%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F331%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1101%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1013%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F400%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1700%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2430%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F344%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F312%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1801%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1303%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2429%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F11%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F325%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2600%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2102%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F703%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1007%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2424%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F338%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F306%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1606%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F320%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1500%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2437%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F200%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1002%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F319%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F333%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F301%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1601%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1015%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F900%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2432%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F346%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F314%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1803%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1010%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F327%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F341%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F705%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1009%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1300%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2426%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2440%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F308%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1608%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F322%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F700%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2439%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1004%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2200%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2421%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F335%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F303%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1603%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2301%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1017%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2434%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F316%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F330%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1805%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1100%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1012%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F329%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F343%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F311%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F500%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1800%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1302%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2428%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2442%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1901%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F324%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2101%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F702%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1006%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2423%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F337%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F305%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1605%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1019%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2436%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1001%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F318%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F332%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1600%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1102%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1014%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2431%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F345%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F313%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1802%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2500%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F326%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F340%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F704%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1008%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2425%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F339%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1607%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F307%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F100%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1400%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F321%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2438%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1003%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2420%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F334%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F800%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1602%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F302%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2300%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1016%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2433%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F347%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F315%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1804%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1011%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1200%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F328%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F342%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F310%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F706%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1301%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2427%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2441%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F309%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F600%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1900%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F323%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2100%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F701%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1005%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2422%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F336%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F304%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1604%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F350%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2302%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F1018%3E%0A%3Chttps%3A%2F%2Fgeo.ld.admin.ch%2Fbiota%2Fforestfirewarningregion%2Fgeometry%2F2435%3E%0A++++++++++++%7D%0A++++++%0A++++++++++++%3Fgeometry+geo%3AasWKT+%3FwktString+.%0A%7D%0A%0A%0A%0A&contentTypeConstruct=text%2Fturtle&contentTypeSelect=application%2Fsparql-results%2Bjson&endpoint=https%3A%2F%2Fgeo.ld.admin.ch%2Fquery&requestMethod=POST&tabTitle=Query&headers=%7B%7D&outputFormat=table)
@bprusinowski any plausible explanation about this being related to visualize code? i.e. based on how the query time is measured ...

**Comment by bprusinowski** [comment]
Hi @Rdataflow, I checked the file where we query geo shapes and the last change (that improved performance a lot) was done in this commit: https://github.com/visualize-admin/visualization-tool/commit/249d1d530b14a97e90ca0a02e2d064d30ec46f12, which was included in 3.21.1 version, so it's also there on PROD.

I did not find anything related to the code / way of measuring the time that could case such huge decrease in performance. I guess we could see if things improve once we have a new deployment to PROD; but it also might be worth to check the infrastructure somehow. But it's strange that's it's only a single query that's affected 🤔
